stream-chat-android-offline/src/test/java/io/getstream/chat/android/livedata/mockwebserver/SendMessageUseCaseTest.kt

```diff
@@ -0,0 +1,50 @@
+package io.getstream.chat.android.livedata.mockwebserver
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.google.common.truth.Truth
+import io.getstream.chat.android.client.models.Message
+import io.getstream.chat.android.client.models.name
+import io.getstream.chat.android.client.utils.SyncStatus
+import io.getstream.chat.android.livedata.utils.isRequestBodyEqualTo
+import kotlinx.coroutines.runBlocking
+import okhttp3.mockwebserver.MockResponse
+import okhttp3.mockwebserver.RecordedRequest
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+internal class SendMessageUseCaseTest : BaseMockWebServerIntegrationTest() {
+
+    override fun dispatchApiRequest(request: RecordedRequest): MockResponse {
```

Comment:
How are we going to handle a case when inside one test class we want to check what happens when a message is sent successfully and what happens when a request fails?
Maybe it would be a good idea to replace this abstract method with the property that will allow configuring responses inside each test method?